linode/manager
# M3-6013: Resolve `loader-utils` Dependabot

## Description 📝

- Updates packages that use `loader-utils` **but** also adds a yarn resolution because many packages we depend on still use a very old version of it
  - See https://github.com/storybookjs/storybook/pull/19891#issuecomment-1349582334 for example
- Fixes some unused imports in some Storybook stories
- Pins `@types/react` to `17.x.x` using Yarn resolution so we can regenerate our yarn.lock without yarn pulling `@types/react@18`
  - https://github.com/facebook/react/issues/24304
-  Forces `tsup` to use an older version because newer versions use rollup 3 which do not allow you to use our version node 14

## `yarn why loader-utils`
These are the packages blocking us from **not** using a yarn resolution 
- `@storybook/builder-webpack4#loader-utils@1.4.2`
- `@storybook/manager-webpack4#loader-utils@1.4.2`
- `babel-loader#loader-utils@1.4.2`
- `webpack#loader-utils@1.4.2`
- `css-loader#loader-utils@1.4.2`

## How to test 🧪

- Test Storybook
- Test Cloud Manager
- Make sure preview link builds and works
  - This helps us prove that a production build will work
